PowerToys.sln

```diff
@@ -520,8 +520,8 @@ Global
 		{89F34AF7-1C34-4A72-AA6E-534BCF972BD9}.Release|x64.ActiveCfg = Release|x64
 		{89F34AF7-1C34-4A72-AA6E-534BCF972BD9}.Release|x64.Build.0 = Release|x64
 		{89F34AF7-1C34-4A72-AA6E-534BCF972BD9}.Release|x86.ActiveCfg = Release|x64
-		{2BE46397-4DFA-414C-9BD4-41E4BBF8CB34}.Debug|ARM64.ActiveCfg = Debug|ARM64
-		{2BE46397-4DFA-414C-9BD4-41E4BBF8CB34}.Debug|ARM64.Build.0 = Debug|ARM64
+		{2BE46397-4DFA-414C-9BD4-41E4BBF8CB34}.Debug|ARM64.ActiveCfg = Debug|x64
```

Comment:
Starting with this and all changes bellow are ARM64->X64? Is that ok?

Reply:
I'm glad you caught this. Those projects shouldn't have been set back to x64. I have a feeling it happened by VS, since I changed it in the UI rather than replacing the string manually

Reply:
@stefansjfw  - I've made commits to resolve that problem.